contrib/kube-prometheus/docs/How-to-monitor-an-external-secure-etcd-cluster.md

```diff
@@ -0,0 +1,164 @@
+# How to monitor a secure external etcd service with Prometheus Operator
```

Comment:
We can probably cut this down to "Monitoring external etcd", same for the file name.